sample_app/app/assets/stylesheets/colors.scss

```diff
@@ -3,5 +3,5 @@ $gray: #eaeaea;
 $light-gray: #777;
 $medium-light-gray: #555;
 $high-light-gray: #bbb;
-$red: #ff0;
+$red: #ff0000;
```

Comment:
:v ờ, nếu viết gọn lại như cách cũ của em thì nó thành màu vàng là đúng oài

Em có thể tham khảo một vài bài viết về cách viết rút gọn mã màu thập lục phân trong css nhé:
http://www.websiteoptimization.com/speed/tweak/hex/
https://stackoverflow.com/questions/3108860/how-to-use-3-digit-color-codes-rather-than-6-digit-color-codes-in-css